### DISCUSSION TITLE
可以把这些功能集成到 Bilibili Evolved 里面吗？

### DISCUSSION THREAD

**Discussion by ChaXxl** [BewlyBewly/BewlyBewly#484] [Ideas]
如果可以集成到 Bilibili Evolved 里面，这样就不用装插件了

**Comment by hakadao** [comment]
我覺得如果有人可以將bilibili evolved可以集成到bewlybewly裏，這樣你就不用安裝個油猴再裝個腳本了，直接裝bewlybewly這個插件就可以了
